src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/SimpleRenameStrategy.cs

```diff
@@ -19,11 +19,11 @@ public SimpleRenameStrategy(IProjectThreadingService threadingService, IUserNoti
         }
 
         // For the SimpleRename, it can attempt to handle any situtation
-        public override bool CanHandleRename(string oldFileName, string newFileName)
+        public override bool CanHandleRename(string oldFileName, string newFileName, bool IsCaseSensitive)
         {
             var oldNameBase = Path.GetFileNameWithoutExtension(oldFileName);
             var newNameBase = Path.GetFileNameWithoutExtension(newFileName);
-            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase);
+            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase) && (string.Compare(Path.GetFileName(oldNameBase), Path.GetFileName(newNameBase), IsCaseSensitive) !=0);
```

Comment:
@jinujoseph Can you fix this to not call into C# specific SyntaxFacts? You should create a C# and VB specific rename strategy in corresponding layers and use language specific SyntaxFacts in those projects.